packages/backend/src/config.mjs

```diff
@@ -33,6 +33,8 @@ export const CONTRIBUTION_TIMEOUT = +(
 )
 export const PRUNE_INTERVAL = +(process.env.PRUNE_INTERVAL ?? 12 * 1000)
 
+export const ENDS_AT = +(process.env.ENDS_AT ?? 10 ** 13)
```

Comment:
Why `10**13` here? or it just a random number?
For the timestamp for Dec 12 2023 here ⬇️
![image](https://github.com/Unirep/ceremony/assets/9023842/7fe02262-7fa9-4c0c-a535-085d40d95bba)

Reply:
Also use this in frontend as default `ENDS_AT` timestamp